homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
```

Comment:
FYI: coordinator.py:71 `data = PlugwiseData(devices={}, gateway={})` doesn't do anything

Reply:
If I remember correctly this init was needed to keep `mypy` happy.

Reply:
Reviewing the code, it's because we try-else the async_add_remove with the variable, created issue on our list to pic this up (upcoming PR will still flag in quality_scale.yaml to track)

Reply:
Amended to #132175